hubblestack/status.py

```diff
@@ -598,9 +598,9 @@ def _setup_for_testing():
     global __opts__
     import hubblestack.daemon
     parsed_args = hubblestack.daemon.parse_args()
-    import salt.config
+    import hubblestack.config
     parsed_args['configfile'] = config_file = '/etc/hubble/hubble'
-    __opts__ = salt.config.minion_config(config_file)
+    __opts__ = hubblestack.config.get_config(config_file)
     __opts__['conf_file'] = config_file
     __opts__.update(parsed_args)
     import salt.loader
```

Comment:
should we change it to hubblestack.loader instead?

Reply:
I intend to do a huge cleanup wave for stuff like that. You're totally right of course, but I don't think it's necessary just now.

Reply:
Yeah, I went ahead and changed this. It only lives in an unused setup and testing function ... or, at least, I doubt anyone's used it since I developed that module.